src/main/java/org/apache/maven/buildcache/xml/CacheConfigImpl.java

```diff
@@ -452,7 +453,9 @@ public boolean isEnabled() {
     @Override
     public boolean isRemoteCacheEnabled() {
         checkInitializedState();
-        return getRemote().getUrl() != null && getRemote().isEnabled();
+        return getRemote().getUrl() != null
+                && Boolean.parseBoolean(System.getProperty(
```

Comment:
No, use properties from the session.

Reply:
Thanks!  I went ahead and updated a couple other Boolean.getProperty(..) methods as those also go to the system properties.